packages/playground/src/components/node_selector/select_location_internals/TfSelectCountry.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <VAutocomplete label="Region" placeholder="Select a region" v-bind="$props.countryProps" clearable />
```

Comment:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/8729a8f7-cf98-4183-a537-e671c32eb1b6)


```suggestion
  <VAutocomplete label="Country" placeholder="Select a region" v-bind="$props.countryProps" clearable />
```